tests/hmm/run.sh

```diff
@@ -20,7 +20,11 @@ export KERAS_BACKEND=tensorflow
 
 #This is the location of the input NanoAOD and generally does not need to be changed
 export INPUTDATAPATH=/storage/group/allcit
+
+#nanoAODv5
 export CACHEPATH=/storage/user/jpata/hmm/skim_merged
+#nanoAODv6
+#export CACHEPATH=/storage/user/nlu/hmm/skim_merged
```

Comment:
I think we can change the default to the private nanoAODv6 now and take this as the new baseline?